README.md

```diff
@@ -1 +1,146 @@
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Portfolio - Mobile Section] <a name="about-project"></a>
+
+**[Portfolio - Mobile Section]** is a great website tool that helps to secure a job as a web developer,
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>HTML, CSS</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Github flow</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">Express.js</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>VScode</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+Install a code editor
+
+### Setup
+
+Clone this repository to your desired folder:
+
+cd my-folder
+
+git clone repo-link
+
+### Install
+
+Install this project with:
+
+### Usage
+
+To run the project, execute the following command:
+
+Press ctrl+shif+p on vscode and choose run
+
+### Run tests
+
+To run tests, run the following command:
+
+Press ctrl+shift+p on vscode and choose test
+
+### Deployment
+
+You can deploy this project using:
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="Michael"></a>
+
+👤 **Author1**
+
+
+- GitHub: [@githubhandle](https://github.com/Mike47ip)
+- Twitter: [@twitterhandle](https://twitter.com/Mikepee47)
+- LinkedIn: [@LinkedIn](https://www.linkedin.com/in/michael-darkwah-81a039141/)
+- Facebook: [@facebook](https://web.facebook.com/profile.php?id=100089565852279)
+
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[My Projects]**
+- [ ] **[About Me]**
+- [ ] **[Contact]**
+
+
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project show your love by leaving a comment
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank all Micronauts for their support
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](https://github.com/Kwaku28/Portfolio-Mobile/blob/toolbar-headline-section/MIT.md) licensed.
+
+<!-- Contributing -->
+If you want to contribute to my project and make it better, your help is very welcome. Contributing is also a great way to learn more about social coding on Github, new technologies and and their ecosystems and how to make constructive, helpful bug reports, feature requests and the noblest of all contributions: a good, clean pull request.
```

Comment:
- [ ] Please make sure to follow the [Microverse template](https://github.com/microverseinc/readme-template) when adding the "Contributing" section as I noticed you've added the contributing text In the MIT section this makes your project look unprofessional and a bit reckless. It should instead be added in it's own section with the proper heading as shown below 👇
  ![image](https://github.com/Mike47ip/mobilePortfolioWeb/assets/104968236/1ae36119-e713-4b5a-a384-21cc1c3919bf)